hammers/scripts/lease_stack_reaper.py

```diff
@@ -98,17 +97,18 @@ def find_stacked_leases(self, leases):
 
         for i in range(len(leases)):
 
-            _, start_date, end_date = leases[i]
```

Comment:
What is this syntax? Does it assume the dict is ordered precisely? Or does it do some sort of dict expansion where the variable names must match the dict keys? (Not super relevant b/c it's gone, just wondering.)

Reply:
leases were originally a tuple in the User class so this was just tuple expansion.